x-pack/solutions/security/plugins/cloud_security_posture/public/pages/configurations/findings_flyout/findings_flyout.tsx

```diff
@@ -112,20 +108,22 @@ export const CspFlyoutMarkdown: React.FC<PropsOf<typeof EuiMarkdownFormat>> = (p
 export const BenchmarkIcons = ({
   benchmarkId,
   benchmarkName,
+  size = 'xl',
 }: {
   benchmarkId: BenchmarkId;
   benchmarkName: BenchmarkName;
+  size?: 's' | 'm' | 'l' | 'xl' | 'original' | 'xxl' | undefined;
```

Comment:
Since this property is to be used by the `EuiIcon`, it is better to point directly to the EuiIconProps size type, so it stays in sync in case Eui changes:

```suggestion
  size?: EuiIconProps['size'];
```

Also, since it's setting `size` as an optional property, it already accounts for `undefined, ' so there's no need to include it explicitly